front/src/applications/operationalStudies/components/ImportTrainSchedule/ImportTrainScheduleModal.jsx

```diff
@@ -7,7 +7,7 @@ import { useDispatch, useSelector } from 'react-redux';
 import { getRollingStockID, getInfraID, getTimetableID } from 'reducers/osrdconf/selectors';
 import generatePathfindingPayload from 'applications/operationalStudies/components/ImportTrainSchedule/generatePathfindingPayload';
 import generateTrainSchedulesPayload from 'applications/operationalStudies/components/ImportTrainSchedule/generateTrainSchedulesPayload';
-import getTimetable from 'applications/operationalStudies/components/Scenario/getTimetable';
+import getTimetable from 'applications/operationalStudies/components/Scenario/getSimulationResults';
```

Comment:
You need to change the name of the function in the file too :)
```suggestion
import getSimulationResults from 'applications/operationalStudies/components/Scenario/getSimulationResults';
```